DQM/L1TMonitor/src/L1TStage2CPPF.cc

```diff
@@ -0,0 +1,536 @@
+#include "DQM/L1TMonitor/interface/L1TStage2CPPF.h"
+#include "DQM/RPCMonitorDigi/interface/utils.h"
+
+using namespace std;
+using namespace edm;
+
+L1TStage2CPPF::L1TStage2CPPF(const ParameterSet& ps)
+    : rpcdigiSource_(ps.getParameter<InputTag>("rpcdigiSource")),
+      rpcdigiSource_token_(consumes<RPCDigiCollection>(ps.getParameter<InputTag>("rpcdigiSource"))),
+      cppfdigiSource_token_(consumes<l1t::CPPFDigiCollection>(ps.getParameter<InputTag>("rpcdigiSource"))) {
+  // verbosity switch
+  verbose_ = ps.getUntrackedParameter<bool>("verbose", false);
+
+  if (verbose_)
+    cout << "L1TStage2CPPF: constructor...." << endl;
```

Comment:
@cobby319 please do not use ```cout``` in module that should run in production, as it is not thread-safe. Use the ```MessageLogger``` instead